public/data/CM-Sharan.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "C M SHARAN",
+  "type": "personal",
+  "bio": "Full Stack Developer And Open Source Enthusiast!",
+  "avatar": "https://github.com/CM-Sharan.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/CM-Sharan",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/cm_sharan",
+      "icon": "twitter"
+    }
```

Oops, we missed this.
```suggestion
    }
  }
```